database/frontend/illustrations.json

```
@@ -1,50 +1,86 @@
-[{
-        "name": "Glaze",
-        "description": "All in one resource to get free illustrations images for characters, backgrounds, avatar and more",
-        "url": "https://glazestock.com/",
-        "category": "frontend",
-        "subcategory": "illustrations"
-    },
-    {
-        "name": "unDraw",
-        "description": "unDraw os another exccelent platform to get free illustration images with No color restrictions",
-        "url": "https://undraw.co/illustrations",
-        "category": "frontend",
-        "subcategory": "illustrations"
-    },
-    {
-        "name": "DrawKit",
-        "description": "It is a collection of fully customisable PSD illustrations, in two styles, available to use on your next project.",
-        "url": "https://drawkit.com/",
-        "category": "frontend",
-        "subcategory": "illustrations"
-    },
-    {
-        "name": "ManyPixels",
-        "description": "Many pixel illustration 2500+ royalty free illustrations to power up your designs",
-        "url": "https://www.manypixels.co/gallery/",
-        "category": "frontend",
-        "subcategory": "illustrations"
-    },
-    {
-        "name": "Reshot",
-        "description": "Free Icons & Illustrations design freely with instant downloads and commercial licenses.",
-        "url": "https://www.reshot.com/",
-        "category": "frontend",
-        "subcategory": "illustrations"
-    },
-    {
-        "name": "StorySet",
-        "description": "Awesome free customizable illustrations for your next project",
-        "url": "https://storyset.com/",
-        "category": "frontend",
-        "subcategory": "illustrations"
-    },
-    {
-        "name": "Freepik",
-        "description": "Freepik goal is to offer the best high-quality content: illustrations, photos, icons, mockups, and presentations templates etc.",
-        "url": "https://www.freepik.com/",
-        "category": "frontend",
-        "subcategory": "illustrations"
-    }
+[
+  {
+    "name": "Glaze",
+    "description": "All in one resource to get free illustrations images for characters, backgrounds, avatar and more.",
+    "url": "https://glazestock.com/",
+    "category": "frontend",
+    "subcategory": "illustrations"
+  },
+  {
+    "name": "unDraw",
+    "description": "unDraw os another exccelent platform to get free illustration images with No color restrictions.",
+    "url": "https://undraw.co/illustrations",
+    "category": "frontend",
+    "subcategory": "illustrations"
+  },
+  {
+    "name": "DrawKit",
+    "description": "It is a collection of fully customisable PSD illustrations, in two styles, available to use on your next project.",
```

[Proofreading] What are the two styles that you mentioned? I think adding this information could help developers understand why they should use this source. :)  
```suggestion
    "description": "This is a collection of fully customizable PSD illustrations. They are done in two styles, and are available to use in your next project.",
```